### PR TITLE
+session.0.4.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

**Comment by samoht** [comment]
Can you remove the `name` field in the opam files? They don't correspond to the package names.

File: packages/session-cohttp-async/session-cohttp-async.0.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "session"
```
Comment (avsm) [inline packages/session-cohttp-async/session-cohttp-async.0.4.0/opam]: The `name` fields in these PRs are wrong; they can just be removed as they are inferred in the main repository.

**Comment by seliopou** [comment]
The Travis build is OS X timing out and the rest seem to be related to transient network failures.

**Comment by avsm** [comment]
Thanks and apologies for the delay; we are working through a backlog of PRs! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**